extension/goplugin/schemas_test.go

```diff
@@ -154,32 +154,56 @@ var _ = Describe("Schemas", func() {
 	Context("Properties", func() {
 		It("Should get correct properties", func() {
 			Expect(testSchema.Properties()).To(Equal(
```

Comment:
Nit: not a big fan of checking complex maps using one expectation.

Reply:
what do you suggest instead?

Reply:
To check each "subtree" separately. That way we won't have to decode and compare large portions of text if something fails.
If you want to be sure that map is equal to sum of those "subtrees" and there are no unexpected ones, just check it's key count.

Reply:
there you go